podcasts/MainTabBarController.swift

```diff
@@ -189,6 +189,7 @@ class MainTabBarController: UITabBarController, NavigationProtocol {
 
         if FeatureFlag.encourageAccountCreation.enabled,
            !Settings.hasShownInformationalViewModal,
+           Settings.hasSeenInitialOnboardingBefore == true,
```

Comment:
`Settings.hasSeenInitialOnboardingBefore` can we avoid the `== true`?

Also does this logic apply when the new onboarding FF is off?